src/azure-cli/azure/cli/command_modules/storage/_params.py

```diff
@@ -233,10 +233,13 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
                    arg_type=get_three_state_flag(),
                    help='A boolean indicating whether or not the service applies a secondary layer of encryption with '
                    'platform managed keys for data at rest.')
-        c.argument('allow_blob_public_access', arg_type=get_three_state_flag(), min_api='2019-04-01', is_preview=True,
+        c.argument('allow_blob_public_access', arg_type=get_three_state_flag(), min_api='2019-04-01',
```

Comment:
Is it better to define a CLIArgumentType for allow_blob_public_access to reuse as the help is very long? also for min_tls_version